tests/kuttl/tests/basic-deployment/02-update-extramounts-horizon.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: |
+      touch mycustompolicy.yaml
+      oc create configmap -n $NAMESPACE horizon-policy --from-file mycustompolicy.yaml
```

Comment:
is the `mycustompolicy.yaml` file already present in `tests/kuttl/tests/basic-deployment` ?
It's my only question, otherwise looks good to me!